tests/server/csp.js

```diff
@@ -66,7 +66,6 @@ suite.tests['blockingRules'] = function () {
   assert.lengthOf(imgSrc, 5);
```

Comment:
Since you've removed an item from the list, this code that's checking the length of the list will also need to change to reflect the new length of the list.